include/mbedtls/config.h

```diff
@@ -2305,22 +2305,26 @@
 /**
  * \def MBEDTLS_THREADING_C
  *
- * Enable the threading abstraction layer.
+ * Enable the threading abstraction layer, to enable use of mutexes within
```

Comment:
I would suggest changing `to enable use of mutexes within mbed TLS` into `to use mbed TLS thread-safe features`.